core/src/main/java/org/bitcoinj/crypto/ECKey.java

```diff
@@ -1247,9 +1247,8 @@ public boolean isEncrypted() {
         return keyCrypter != null && encryptedPrivateKey != null && encryptedPrivateKey.encryptedBytes.length > 0;
     }
 
-    @Nullable
     @Override
-    public Protos.Wallet.EncryptionType getEncryptionType() {
+    public Protos.Wallet.@Nullable EncryptionType getEncryptionType() {
```

Comment:
This seems weird. How does this even compile? I'd expect

`@Nullable public Protos.Wallet.EncryptionType…`

Reply:
The new annotations are very specific at where in the "hierarchy" the nullability exists. This type of annotation is supported by the Java compiler.